game/scripts/npc/heroes/treant.txt

```diff
@@ -0,0 +1,12 @@
+"DOTAHeroes"
+{
+  //=================================================================================================================
+  // HERO: Treant Protector
+  //=================================================================================================================
+  "npc_dota_hero_treant"
+  {
+    "Ability10"                                           "special_bonus_unique_treant_9"
```

Comment:
I am not a fan of having 2 talents upgrading the same ability at the same level. In this case 2 Nature's Grasp talents.